MicrowaveGame/Assets/Resources/Scripts/Menus/MenuBehaviour.cs

```diff
@@ -7,6 +7,11 @@ namespace Scripts.Menus
 	/// </summary>
 	public abstract class MenuBehaviour : MonoBehaviour
 	{
+		/// <summary>
+		/// SceneFader gameobject
+		/// </summary>
+		public SceneFader sceneFader;
```

Comment:
`sceneFader` should be renamed to `sceneFaderBehaviour` to match the class name (class name change discussed in another comment).

Additionally, the access modifier should be `protected` since it's only used inside of classes that extend `MenuBehaviour`.